kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/js/kieserver-ui.js

```diff
@@ -60,6 +60,16 @@ function startProcess(button) {
 	}
 }
 
+function getParentCorrelationKey() {
```

Comment:
whitespace formatting, please add spaces before the if opening brackets.

Reply:
I think the function's name is not ideal, I would suggest sth as `resolveCorrelationKey` instead. Because it gets correlation key either from itself or its parent, but I agree that at the same time there's the conflict in naming (getCorrelationKey being already a defined term).

Reply:
Done